core/src/main/resources/org/apache/spark/ui/static/spark-dag-viz.js

```diff
@@ -209,6 +209,12 @@ function renderDagVizForJob(svgContainer) {
     // Actually render the stage
     renderDot(dot, container);
 
+    // Round corners on rectangles
+    container
+      .selectAll("rect")
+      .attr("rx", "4")
+      .attr("ry", "4");
```

Comment:
you could put this in `renderDagViz` itself to reduce the duplication, since we do it for stages too